dmsguide.md

```diff
@@ -602,22 +602,22 @@ A tunic woven from the hair of a spriggan. While outside you have expertise in s
 - **AP Cost:** 5
 - **Range:** 7
 
-When standing next to a tree you may magically enter the tree. While inside the tree you are untargetable by effects that do not target the ethereal plane. At the start of your next turn you appear beside a tree within range.
+When standing next to a tree you may magically enter the tree. While inside the tree you are un-targetable by effects that do not target the ethereal plane. At the start of your next turn you appear beside a tree within range.
```

Comment:
Everything in these changes are great, although we probably need to change all that 'un-'s back to 'un'. Untargetable is a valid word, at least in the gaming world. Even though it has big red squigglies as I'm writing this.